src/assets/wise5/authoringTool/peer-grouping/select-peer-grouping-dialog/select-peer-grouping-dialog.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Inject, OnInit } from '@angular/core';
+import { MatDialog, MatDialogRef, MAT_DIALOG_DATA } from '@angular/material/dialog';
+import { PeerGrouping } from '../../../../../app/domain/peerGrouping';
+import { PeerGroupingAuthoringService } from '../../../services/peerGroupingAuthoringService';
+import { CreateNewPeerGroupingDialogComponent } from '../create-new-peer-grouping-dialog/create-new-peer-grouping-dialog.component';
+
+@Component({
+  selector: 'select-peer-grouping-dialog',
+  templateUrl: './select-peer-grouping-dialog.component.html',
+  styleUrls: ['./select-peer-grouping-dialog.component.scss']
+})
+export class SelectPeerGroupingDialogComponent implements OnInit {
+  peerGroupings: PeerGrouping[] = [];
+
+  constructor(
+    private dialog: MatDialog,
+    private dialogRef: MatDialogRef<SelectPeerGroupingDialogComponent>,
+    @Inject(MAT_DIALOG_DATA) private selectedTag: string,
```

Comment:
```suggestion
    @Inject(MAT_DIALOG_DATA) private selectedPeerGrouping: PeerGrouping,
```

Should we be working with PeerGrouping objects in SelectPeerGroupingDialogComponent and the dialog components that are launched from this class?